config/jobs/kubernetes/wg-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -49,6 +49,27 @@ periodics:
         else
           echo "Scan completed"
         fi
+
+        # snyk scan on container images
+        echo "Fetch the list of k8s images"
+        curl -L https://sbom.k8s.io/v1.22.1/release | grep 'PackageName: k8s.gcr.io/' | awk '{print $2}' > images
+        while read image; do
+          echo "Running snyk scan .."
```

Comment:
It might be useful to update the echo statements here and in the original script to reflect the type of scanning that is ongoing. e.g. `Running build time deps scan ...` or `Running container image scan`

Reply:
Done.